src/GeniusYield/Types/Logging.hs

```diff
@@ -261,7 +274,7 @@ cfgAddContext i cfg = cfg { cfgLogContexts = addContext i (cfgLogContexts cfg) }
 logRun :: (HasCallStack, MonadIO m, StringConv a Text) => GYLogConfiguration -> GYLogSeverity -> a -> m ()
 logRun GYLogConfiguration {..} sev msg = case cfgLogDirector of
   Left cfgLogEnv -> withFrozenCallStack $ K.runKatipT (logEnvToKatip cfgLogEnv) $ K.logLoc (logContextsToKatip cfgLogContexts) (logNamespaceToKatip cfgLogNamespace) (logSeverityToKatip sev) (K.logStr msg)
-  Right GYRawLog {..} -> liftIO $ rawLogRun (toS @Text @String $ "[Namespaces: " <> Text.intercalate ", " (K.unNamespace $ logNamespaceToKatip cfgLogNamespace) <> "]" <> "[Contexts: " <> logContextsToS cfgLogContexts <> "] " <> toS @_ @Text msg)
+  Right GYRawLog {..} -> liftIO $ unRawLogger rawLogRun cfgLogContexts cfgLogNamespace sev $ toS msg
```

Comment:
It appears that this change no longer shows contexts & namespaces for privnet logs.

Reply:
Ah, I think that should be added to `RawLogger`. I agree that it should exist.

Reply:
Actually I think this should still be able to show contexts and namespaces but it depends on the `GYRawLog`. So it's now controlled higher up the call stack. Specifically, it will depend on the `GYLogConfiguration` and how it was obtained. The pre-provided `simpleRawLogger` does not log them. (I found them to be extra noise in privnet tests).

Feel free to add a more verbose `RawLogger`!